pandas/tests/test_graphics.py

```diff
@@ -1319,18 +1320,23 @@ def test_subplots_multiple_axes(self):
         # (show warning is tested in
         # TestDataFrameGroupByPlots.test_grouped_box_multiple_axes
         fig, axes = self.plt.subplots(2, 2)
-        df = DataFrame(np.random.rand(10, 4),
-                       index=list(string.ascii_letters[:10]))
-
-        returned = df.plot(subplots=True, ax=axes, layout=(2, 1))
-        self._check_axes_shape(returned, axes_num=4, layout=(2, 2))
-        self.assertEqual(returned.shape, (4, ))
-
-        returned = df.plot(subplots=True, ax=axes, layout=(2, -1))
-        self._check_axes_shape(returned, axes_num=4, layout=(2, 2))
-        self.assertEqual(returned.shape, (4, ))
-
-        returned = df.plot(subplots=True, ax=axes, layout=(-1, 2))
+        with warnings.catch_warnings():
+            warnings.simplefilter('ignore')
```

Comment:
Required, because the test specifies `layout` keyword which can't be effective when `ax` keyword is passed. I understand these tests was added in #8297, and intend to test `layout` is ignored properly.

@TomAugspurger ?